tools/tuf/test/docker_package.sh

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Nit: Are shebangs required/good-practice these days?

Also please add a small comment around the purpose of this script.